services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,

```

Comment:
It is Optional[list[int]] it accepts None by default
Do not forgot add inner annotations for list

Reply:
Fix in other places